ResearchKit/Consent/ORKConsentSignatureFormatter.h

```diff
@@ -0,0 +1,8 @@
+#import <ResearchKit/ResearchKit.h>
+#import <ResearchKit/ORKConsentSignature.h>
+
+@interface ORKConsentSignatureFormatter : NSObject
+
+- (NSString *)HTMLForSignature:(ORKConsentSignature *)signature;
```

Comment:
To add to that - in general, using separate, composable objects for this makes things more testable since you can swap or nil out parts of the implementation easily at runtime rather than having to subclass the class under test multiple times. If you used a class method you'd still need a way to override what class is used, which would require subclassing to override (like, for example, the `layerClass` class method on UIView).